### PR TITLE
Update 0.149 release notes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

**Comment by electrum** [comment]
Keep this open -- I'll add some shortly

File: presto-docs/src/main/sphinx/release/release-0.149.rst [hunk 1]
```diff
@@ -4,10 +4,37 @@ Release 0.149
 
 General Changes
 ---------------
-* ``query.max-age`` property was renamed to ``query.min-expire-age``.
+
+* Fix planning bug that causes queries that use grouping sets over UNIONs to fail.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.149.rst]:
```
* Fix planning failure for queries that use grouping sets over unions.

File: presto-docs/src/main/sphinx/release/release-0.149.rst [hunk 2]
```diff
@@ -4,10 +4,37 @@ Release 0.149
...
+* Fix issue that caused ``ARRAY_AGG`` to ignore null values.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.149.rst]: * Do not ignore null values in ``array_agg``.

File: presto-docs/src/main/sphinx/release/release-0.149.rst [hunk 3]
```diff
@@ -4,10 +4,37 @@ Release 0.149
...
+* Fix failure when casting row values that contain null fields.
+* Fix failure when using complex types as map keys.
+* Fix potential memory tracking leak when queries are cancelled.
+* Fix rejection of queries that do not match any queue/resource group rules.
+  Previously, a 500 error was returned to the client.
+* Fix :func:`trim` and :func:`rtrim` functions to produce more intuitive results
+  when the argument contains invalid ``UTF-8`` sequences.
+* Add a new web interface, with cluster overview, realtime stats and improved sorting
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.149.rst]:
```
* Add a new web interface with cluster overview, realtime stats, and improved sorting
  and filtering of queries.

File: presto-docs/src/main/sphinx/release/release-0.149.rst [hunk 4]
```diff
@@ -4,10 +4,37 @@ Release 0.149
...
+  1970 and 2069 inclusive.
+* Improve memory tracking in exchanges to avoid running out of Java heap space.
+* Improve performance of subscript operator for the ``MAP`` type.
+* Improve performance of ``JOIN`` and ``GROUP BY`` queries.
+
+Hive Changes
+------------
+
+* Clean up empty staging directories after ``INSERT INTO`` Hive table.
+* Add ``hive.ipc-ping-interval`` config for HDFS.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.149.rst]: This is a bad config name. It is specific to HDFS, so it should be named `hive.dfs.ipc-ping-interval`

File: presto-docs/src/main/sphinx/release/release-0.149.rst [hunk 5]
```diff
@@ -4,10 +4,37 @@ Release 0.149
...
+* Change ``%y`` (2-digit year) in :func:`date_parse` to evaluate to a year between
+  1970 and 2069 inclusive.
+* Improve memory tracking in exchanges to avoid running out of Java heap space.
+* Improve performance of subscript operator for the ``MAP`` type.
+* Improve performance of ``JOIN`` and ``GROUP BY`` queries.
+
+Hive Changes
+------------
+
+* Clean up empty staging directories after ``INSERT INTO`` Hive table.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.149.rst]: "Hive table" is not needed here:

```
* Clean up empty staging directories after inserts.

**Comment by electrum** [comment]
General:

```
* Add ``queued`` flag to ``StatementStats`` in REST API.
* Improve error messages for math operations.
```

Hive:
```
* Fix force local scheduling for S3 or other non-HDFS file systems.
```

File: presto-docs/src/main/sphinx/release/release-0.149.rst (hunk 4 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.149.rst]: https://github.com/prestodb/presto/pull/5520

**Pull request closed without merging**